assets/app/views/settings.html

```diff
@@ -1,119 +1,236 @@
 <div class="content">
   <project-page>
-    <div>
-      <div class="page-header page-header-bleed-right">
-        <delete-link
-          class="pull-right fa-lg"
-          resource-type="project"
-          resource-name="{{project.metadata.name}}"
-          project-name="{{project.metadata.name}}"
-          display-name="{{(project | displayName)}}"
-          alerts="alerts"
-          button-only>
-        </delete-link>
-        <h1>Project Settings</h1>
-      </div>
+    <div class="breadcrumb"></div>
+    <alerts alerts="alerts"></alerts>
+    <div class="tile__">
```

Comment:
Agree about the margin above the edit field, though it keeps the 'Display name:' label from jumping when editing.